src/managers/resource_manager/resource_manager.gd

```diff
@@ -187,68 +189,69 @@ func _ready() -> void:
 
 # TODO: Can be optimized, only run on build/tick event, not every frame
 func _physics_process(delta):
-	for idx in range(0, 6):
+	for idx in range(0, 7):
 		calculate_resource_modifier(idx, population_amount)
 		match idx:
-			RESOURCE_TYPE.POPULATION:
-				pass
-			RESOURCE_TYPE.HOUSING:
-				pass
-			RESOURCE_TYPE.FOOD:
+			EnumAutoload.ResourceType.FOOD:
```

Comment:
The intention of the `ResourceManager` singleton is to handle all resource related changes and signals, moving the `RESOURCE_TYPE` enum out into another "enums-only" singleton seems counter-productive here.

Reply:
I think it easier to see all enum in a central location than go hunting them around. Performance wise it probably not gonna impact much
.

Reply:
Yeah but it's less about performance and more about grouping relevant code together imo - I'm fine with this going in but its something I'll probably want to look into more when we get round to doing a full spring-clean of the codebase in the new year.